swift/README.md

```diff
@@ -1 +1,26 @@
-Curbmap have a Swift 4.x implementation cocoapod located [here](https://github.com/curbmap/OpenLocationCode-swift).
+# Swift Implementations of Open Location Code
+
+## Curbmap
+
+Curbmap have a Swift 4.x and 3.x implementation located [here](https://github.com/curbmap/OpenLocationCode-swift). 
+
+Properties:
+* Can be built as a Framework.
```

Comment:
Is there a difference between this and this line below?

* Can be built as a Framework for use with iOS, macOS, tvOS and watchOS.

If not, can we have them be identical?

Reply:
I will investigate and get back to you. The reason they are not enumerated was simply that I didn't know.

Reply:
Looking at the Curbmap version, I see a single Framework target using an iOS base SDK (whereas the OLC for Swift & ObjC library has separate Framework targets for iOS, macOS, tvOS and watchOS). I have updated the readme to enumerate just iOS for Curbmap, and I hope that the Curbmap team can amend as necessary in case of any inaccuracy, as I don't claim to be the expert on their library.

Side note: when I started working on my Swift implementation back in July 2017, the Curbmap implementation hadn't been published, so I didn't do any analysis before setting out. The two implementations turned out fairly different (and yes, I'm a little slow to update this README).

Reply:
Sounds reasonable, thanks for checking.